src/content-helper/dashboard-widget/components/top-posts.tsx

```diff
@@ -112,13 +141,14 @@ export function TopPosts() {
 			{ filters }
 			{
 				loading ? ( spinner ) : (
-					<ol className="parsely-top-posts">
+					<ol className="parsely-top-posts" style={ { counterReset: 'item ' + ( ( page - 1 ) * TOP_POSTS_DEFAULT_LIMIT ) } }>
 						{ posts.map( ( post: PostData ): JSX.Element =>
 							<TopPostListItem key={ post.id } metric={ metric } post={ post } />
 						) }
 					</ol>
 				)
 			}
+			{ ( posts.length >= 5 || page > 1 ) && navigation }
```

Comment:
Can we replace `5` with `TOP_POSTS_DEFAULT_LIMIT` here? If not (or you think this could introduce a bug with an update), fine to leave as is.